cmd/gen-go-sample/main.go

```diff
@@ -338,63 +333,63 @@ func (g *generator) getInitInfo(inType pbinfo.ProtoType, methConf GAPICMethod, v
 	}
 
 	// Set up request object.
-	for _, def := range valSet.Parameters.Defaults {
-		if err := itree.parseInit(def, g.descInfo); err != nil {
-			return initInfo{}, errors.E(err, "can't set default value: %q", def)
+	for _, reqConf := range reqConfs {
+		if err := itree.parseInit(reqConf.Field, reqConf.Value, g.descInfo); err != nil {
+			return initInfo{}, errors.E(err, "can't set default value: %s=%s", reqConf.Field, reqConf.Value)
 		}
 	}
 
 	// Some parts of request object are from arguments.
-	for _, attr := range valSet.Parameters.Attributes {
-		if attr.ReadFile {
+	for _, reqConf := range reqConfs {
+		if reqConf.ValueIsFile {
 			var varName string
 			var err error
-			if attr.SampleArgumentName != "" {
-				varName = snakeToCamel(attr.SampleArgumentName) + fileContentSuffix
+			if reqConf.InputParameter != "" {
```

Comment:
You have this condition several times. I'd assign it its own variable for clarity: 
```
isSampleParam = (fieldConf.InputParameter != "")
```

Reply:
Done. Changed the local var to `isInputParam` to be consistent with config schema. But I can change to `isSampleParam` too.